sql/catalyst/src/main/scala/org/apache/spark/sql/OutputMode.java

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.catalyst.analysis
+package org.apache.spark.sql;
```

Comment:
before that.... i realize that making this java enum prevents us from having output modes like `UpdateInPlace("key")` in the future. So we have to think about this.